easy-ui-react/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "root": true,
+  "extends": ["@easypost/eslint-config-easy-ui"],
+  "overrides": [
+    {
+      "files": ["src/components/**/*.tsx"],
```

Comment:
should this include every tsx/ts file in `easy-ui-react`?

Reply:
actually i forgot that i modified the main eslint config as part of the responsive props tasks that i think fixes issues you were running into?

the eslint config for the repo is governed by the `eslint-config-easy-ui` repo. in `eslint-config-easy-ui/index.js` i made these changes. with them, you shouldn't need an `.eslintrc` in `easy-ui-react`:

<img width="979" alt="image" src="https://user-images.githubusercontent.com/752942/222487013-02445d91-4afa-425f-b7b7-9d7c0a1d9128.png">

```js
/** @type {import('eslint').Linter.Config} */
module.exports = {
  extends: [
    "turbo",
    "plugin:react/recommended",
    "plugin:react-hooks/recommended",
    "plugin:storybook/recommended",
    "plugin:prettier/recommended",
  ],
  settings: {
    react: {
      version: "detect",
    },
  },
  env: {
    node: true,
    browser: true,
  },
  overrides: [
    {
      files: ["**/*.js", "**/*.jsx"],
      extends: ["eslint:recommended"],
      parserOptions: {
        ecmaVersion: "2021",
        sourceType: "module",
      },
    },
    {
      files: ["**/*.ts", "**/*.tsx"],
      extends: ["plugin:@typescript-eslint/recommended"],
      plugins: ["@typescript-eslint"],
      parser: "@typescript-eslint/parser",
    },
  ],
};
```